tests/benchmark/bloatnet/deploy_bloatnet_simple.py

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env python3
+"""
+CREATE2 deployment script for bloatnet benchmark contracts.
+Uses a factory pattern to deploy contracts at deterministic addresses.
+
+Based on the pattern from EIP-7997, this deploys contracts using CREATE2
+so they can be accessed from any account and reused across tests.
+"""
+
+import argparse
+import os
+import subprocess
+import sys
+from typing import Dict, List, Optional, Tuple
+
+from eth_utils import keccak
+from web3 import Web3
+
+
+class ContractType:
+    """Define contract types for different benchmarks."""
+
+    MAX_SIZE_24KB = "max_size_24kb"
+    SLOAD_HEAVY = "sload_heavy"
+    STORAGE_HEAVY = "storage_heavy"
+    CUSTOM = "custom"
+
+
+CONTRACT_DESCRIPTIONS = {
+    ContractType.MAX_SIZE_24KB: "24KB contracts filled with unique bytecode (standard bloatnet)",
+    ContractType.SLOAD_HEAVY: "Contracts optimized for SLOAD benchmarking",
+    ContractType.STORAGE_HEAVY: "Contracts with pre-initialized storage",
+    ContractType.CUSTOM: "Custom bytecode (provide your own)",
+}
+
+
+def generate_max_size_bytecode(salt: int = 0) -> Tuple[bytes, bytes]:
+    """Generate 24KB contract bytecode for standard bloatnet tests."""
+    # Init code copies runtime bytecode to memory and returns it
+    init_code = bytearray()
+
+    # Init code: PUSH2 size, PUSH1 offset, PUSH1 dest, CODECOPY, PUSH2 size, PUSH1 0, RETURN
+    bytecode_size = 24576
```

Comment:
You can read this constant from `fork: Fork` (provided in tests) and read `fork.max_code_size()` to read the constant from the fork.

Reply:
Modified!